Assets/Scripts/Game Object Definitions/Entity Definitions/ShellCore.cs

```diff
@@ -44,7 +44,16 @@ public float GetPower()
 
     public void AddPower(float power)
     {
-        totalPower += power;
+        if (totalPower + power <= 5000){
```

Comment:
Things like this can really be done in one statement: totalPower = Mathf.Max(totalPower+power, 5000);

Reply:
Also that alert is going to get really annoying so remove it

Reply:
done, sorry for making you babysit my code :P